ui/packages/catalog-ui-search/src/main/webapp/react-component/enum/enum.tsx

```diff
@@ -63,6 +67,7 @@ export default hot(module)(
         if (nextProps.value !== this.props.value) {
           this.state.propertyModel &&
             this.state.propertyModel.setValue([nextProps.value])
+          this.setState({ regionComponentKey: Math.random() }) //Forces re-render of the MarionetteRegionContainer
```

Comment:
so we are using a random number just to cause the state to change here?  Really has no meaning just counting on it to be different?

Reply:
Basically, yes, without it the component will not change so no re-render. But it's also a React component key so there is that.